src/s1reader/s1_burst_slc.py

```diff
@@ -313,6 +313,17 @@ def get_sensing_mid(self):
         d_seconds = 0.5 * (self.shape[0] - 1) * self.azimuth_time_interval
         return self.sensing_start + datetime.timedelta(seconds=d_seconds)
 
+    def get_sensing_end(self):
+        '''Returns sensing end as datetime.datetime object.
+
+        Returns:
+        --------
+        _ : datetime.datetime
+            Sensing end as datetime.datetime object.
+        '''
+        d_seconds = (self.shape[0] - 1) * self.azimuth_time_interval
```

Comment:
Now that you are adding handy parameters, another handy parameter is length and width. :)